### DISCUSSION TITLE
Processing all the Subswath (all the frame)

### DISCUSSION THREAD

**Discussion by Obaida777** [gmtsar/gmtsar#490] [Q&A]
Hello Everyone, 

I want to processing all Subswath for S1A, there is any script, Tutorial.

**Comment by Xiaohua-Eric-Xu** [comment]
For pair-wise processing, use p2p_S1_TOPS_Frame.csh. For stack processing, follow the recipe document, process each individual subswaths to interferogram level, use merge_batch.csh to combine them together and then unwrap with snaphu.csh (or unwrap_parallel.csh).
